.circleci/config.yml

```diff
@@ -9,7 +9,8 @@ jobs:
       - run: sudo apt-get update
       - run: sudo apt-get install shellcheck
       - run: pip3 install pre-commit
-      - run: pre-commit run --all-files --verbose
+      #- run: pre-commit run --all-files --verbose
+      - run: pre-commit run --files `git diff --name-only HEAD master`
```

Comment:
Does this work instead?
`pre-commit run --from-ref origin/HEAD --to-ref HEAD`

It seems more inline with running in CI.

https://pre-commit.com/#usage-in-continuous-integration

Reply:
On circleCI, origin/HEAD should be identical master so that does the same thing I think (in a more elegant way). I missed those options. Will update.